share/github-backup-utils/ghe-rsync

```diff
@@ -7,11 +7,17 @@
 
 set -o pipefail
 
+# Bring in the backup configuration
+cd $(dirname "$0")/../..
+. share/github-backup-utils/ghe-backup-config
+
+opts="$GHE_EXTRA_RSYNC_OPTS"
+
 # Filter vanished file warnings from both stdout (rsync versions < 3.x) and
 # stderr (rsync versions >= 3.x). The complex redirections are necessary to
 # filter stderr while also keeping stdout and stderr separated.
 IGNOREOUT='^(file has vanished: |rsync warning: some files vanished before they could be transferred)'
-(rsync "${@}" 3>&1 1>&2 2>&3 3>&- |
+(rsync $opts "${@}" 3>&1 1>&2 2>&3 3>&- |
```

Comment:
I think `$GHE_EXTRA_RSYNC_OPTS` can be used directly here, no need for the intermediate `$opts` variable.